app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -345,6 +336,10 @@ public void displayCurrentInfo() {
                 currentBgValueText.setPaintFlags(currentBgValueText.getPaintFlags() | Paint.STRIKE_THRU_TEXT_FLAG);
                 dexbridgeBattery.setPaintFlags(dexbridgeBattery.getPaintFlags() | Paint.STRIKE_THRU_TEXT_FLAG);
             } else {
+                if (notificationText.getText().length()==0){
+                    notificationText.setTextColor(Color.WHITE);
+                    notificationText.setText((System.currentTimeMillis()-lastBgreading.timestamp)/(60*1000) + " Minutes ago");
```

Comment:
By the way, if we are here maybe we can also add the delta from the last
bg-reading?

On Mon, Jun 15, 2015 at 3:13 AM, Stephen Black <notifications@github.com>
wrote:

> In app/src/main/java/com/eveningoutpost/dexdrip/Home.java
> <https://github.com/StephenBlackWasAlreadyTaken/xDrip-Experimental/pull/59#discussion_r32386384>
> :
>
> > @@ -345,6 +336,10 @@ public void displayCurrentInfo() {
> >                  currentBgValueText.setPaintFlags(currentBgValueText.getPaintFlags() | Paint.STRIKE_THRU_TEXT_FLAG);
> >                  dexbridgeBattery.setPaintFlags(dexbridgeBattery.getPaintFlags() | Paint.STRIKE_THRU_TEXT_FLAG);
> >              } else {
> > +                if (notificationText.getText().length()==0){
> > +                    notificationText.setTextColor(Color.WHITE);
> > +                    notificationText.setText((System.currentTimeMillis()-lastBgreading.timestamp)/(60*1000) + " Minutes ago");
>
> thoughts on rearanging this a bit so that
>
> (starting on line 328)
>
>  if ((new Date().getTime()) - (60000 * 11) - lastBgreading.timestamp > 0) {
>     notificationText.setText("Signal Missed");
>     ...all that other jazz
> } else {
>      if (notificationText.getText().length() == 0){
>         notificationText.setTextColor(Color.WHITE);
>     }
> }
> notificationText.append("\n" + (System.currentTimeMillis() - lastBgreading.timestamp)/(60*1000) + " Minutes ago");
>
> that way even if we know the signal has been missed we can get how long
> ago the last one was
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/StephenBlackWasAlreadyTaken/xDrip-Experimental/pull/59/files#r32386384>
> .
>